diff -up pdfgrep-1.3.2/pdfgrep.cc.omv~ pdfgrep-1.3.2/pdfgrep.cc
--- pdfgrep-1.3.2/pdfgrep.cc.omv~	2022-09-26 05:21:23.371015451 +0200
+++ pdfgrep-1.3.2/pdfgrep.cc	2022-09-26 05:21:31.764067559 +0200
@@ -188,7 +188,7 @@ int search_in_document(poppler::document
 		char *str_start = &str[0];
 #endif
 		int index = 0;
-		struct match mt = {str_start, str.size()};
+		struct match mt = {str_start, static_cast<int>(str.size())};
 
 		while (!max_count_reached && !regexec(needle, str_start+index, 1, match, 0)) {
 			regmatch_to_match(match[0], index, &mt);
